### PR TITLE
Updated side car versions 

### PR DESCRIPTION
**Pull request by shefali-malhotra**
# Description
added support for k8s 1.21 for MKE-3.5.3
Updated side car versions 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/243 |

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [X] Run unit test

### CODE REVIEW

**Review by bpjain2004 — COMMENTED** [review]
what about sidecar changes for k8s 1.22 and 1.23?

**Review by bpjain2004 — APPROVED** [review]

**Comment by randeepsharma** [comment]
Update the side car version for 1.24 as well.

File: helm/csi-isilon/k8s-1.21-values.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  # "images.registrar" defines the container images used for the csi registrar
+  # container.
+  registrar: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.1
+
+  # "images.resizer" defines the container images used for the csi resizer
+  # container.
+  resizer: k8s.gcr.io/sig-storage/csi-resizer:v1.4.0
+
+  # "images.externalhealthmonitorcontroller" defines the container images used for the csi external health monitor controller
+  # container.
```
Comment (randeepsharma) [inline helm/csi-isilon/k8s-1.21-values.yaml:24]: Missed this line at the end.
externalhealthmonitorcontroller: k8s.gcr.io/sig-storage/csi-external-health-monitor-controller:v0.5.0

Comment (shefali-malhotra) [reply helm/csi-isilon/k8s-1.21-values.yaml:24]: addressed

**Review by randeepsharma — APPROVED** [review]

**Review by prablr79 — APPROVED** [review]

**Pull request merged**